### PR TITLE
Gem changes from 3.15 patch notes

### PR DESCRIPTION
**Pull request by LocalIdentity**
I never want to do this again, 8 hours of pain

### CODE REVIEW

File: src/Data/Skills/act_dex.lua [hunk 1]
```diff
@@ -2158,46 +2156,46 @@ skills["QuickDodge"] = {
...
+		[11] = { 30, 30, cooldown = 2.5, levelRequirement = 40, statInterpolation = { 1, 1, }, cost = { Mana = 11, }, },
+		[12] = { 33, 33, cooldown = 2.5, levelRequirement = 44, statInterpolation = { 1, 1, }, cost = { Mana = 12, }, },
+		[13] = { 36, 36, cooldown = 2.5, levelRequirement = 48, statInterpolation = { 1, 1, }, cost = { Mana = 12, }, },
+		[14] = { 39, 39, cooldown = 2.5, levelRequirement = 52, statInterpolation = { 1, 1, }, cost = { Mana = 13, }, },
+		[15] = { 42, 42, cooldown = 2.5, levelRequirement = 55, statInterpolation = { 1, 1, }, cost = { Mana = 14, }, },
+		[16] = { 45, 45, cooldown = 2.5, levelRequirement = 58, statInterpolation = { 1, 1, }, cost = { Mana = 14, }, },
+		[17] = { 48, 48, cooldown = 2.5, levelRequirement = 61, statInterpolation = { 1, 1, }, cost = { Mana = 15, }, },
+		[18] = { 51, 51, cooldown = 2.5, levelRequirement = 64, statInterpolation = { 1, 1, }, cost = { Mana = 15, }, },
+		[19] = { 54, 54, cooldown = 2.5, levelRequirement = 67, statInterpolation = { 1, 1, }, cost = { Mana = 16, }, },
+		[20] = { 19, 57, cooldown = 2.5, levelRequirement = 70, statInterpolation = { 1, 1, }, cost = { Mana = 16, }, },
```
Comment (holyzeus) [inline src/Data/Skills/act_dex.lua:2178]: 19 -> 57

**Comment by xFleury** [comment]
I can't find Spectral Helix in the gem list.

**Comment by Lexir83** [comment]
Thank you for grinding through those updates so quickly.  Just looking to bring attention to missing Storm Rain and Focus Ballista Support.  I see several other of newly added gems are not yet included as well, but that all appears to be in your crosshairs (identified by "Fix some left out gems" commit?) after you recover from your last 8hr shift lol.  Thank you again, appreciate all the hard work.

**Comment by donyil** [comment]
Version 2.4.0. Missing ALL new Gems.

**Comment by pundm** [comment]
> 
> 
> Version 2.4.0. Missing ALL new Gems.

They will be included on patch day.

**Pull request merged**